### ISSUE TITLE
feat(R): Support LayerNormalization layer in orbital_keras_dag_impl (parity with Python #14)

### ISSUE THREAD

**Issue by davidrsch** [davidrsch/orbital-r-fork#20]
## Summary

`keras.layers.LayerNormalization` is widely used in modern neural network architectures, including transformer-based feature extractors and tabular MLPs. The `orbital_keras_dag_impl()` function introduced in `feat/ann-mlp-support` currently raises an error for unrecognised layer types — `LayerNormalization` is not yet dispatched.

At inference time, `LayerNormalization` applies a deterministic formula over the **feature axis** (last axis by default) using learned gamma and beta parameters:

```
y_i = ((x_i - mean(x)) / sqrt(var(x) + eps)) * gamma_i + beta_i
```

Unlike `BatchNormalization`, LayerNorm computes mean and variance **per row** — the normalisation is computed dynamically over the column expressions in the orbital expression vector. This is fully expressible in standard SQL.

For parity, Python issues [orbital-1 #14](https://github.com/davidrsch/orbital-1/issues/14) (ReduceMean) and [orbital-1 #16](https://github.com/davidrsch/orbital-1/issues/16) (Sqrt, Pow) together enable LayerNorm subgraph translation in the Python package.

## Acceptance Criteria

- `LayerNormalization` layer case handled in `orbital_keras_dag_impl()` dispatch (in `R/model-keras.R`)
- Retrieve per-feature parameters from the keras layer: `gamma` and `beta` vectors via `layer$get_weights()`
- `epsilon` retrieved via `layer$epsilon` (default `1e-3` in keras3)
- Compute row-wise mean and variance over the incoming expression vector:
  - `mean_expr = (expr_1 + expr_2 + ... + expr_n) / n`
  - `var_expr = ((expr_1 - mean_expr)^2 + ... + (expr_n - mean_expr)^2) / n`
  - `norm_i = (expr_i - mean_expr) / sqrt(var_expr + eps)`
  - `out_i = norm_i * gamma_i + beta_i`
- Output stored in the registry under the layer name
- `orbital_keras_impl()` DAG detection updated to also trigger for models containing `LayerNormalization` layers
- Integration test: a simple `Sequential(Dense(relu), LayerNormalization, Dense)` model — verify `orbital()` output matches `predict(model, df)` (skip if keras3 not installed)

## Notes

- keras3 API: `layer$get_weights()` returns `list(gamma, beta)` — both vectors of length `n_features`
- `axis` attribute: LayerNorm over `axis=-1` (the feature/last axis) is the only SQL-translatable case. If `axis` differs, raise an informative error.
- The mean/var are computed **symbolically** over the R expression vector — the resulting SQL will have nested arithmetic expressions but is correct
- For large feature vectors the expression string will be long, but that is documented as a known limitation

**Comment by davidrsch** [comment]
Implemented in commit `46d7140` on `feat/ann-mlp-support`.\n\nChanges in `R/model-keras.R` `orbital_keras_dag_impl()`:\n- **LayerNormalization dispatch**: reads `gamma`, `beta` via `l$get_weights()`; reads `epsilon` via `l$epsilon`; computes symbolic per-row mean (`orbital_ln_mean_{lname}`) and variance (`orbital_ln_var_{lname}`) as intermediate expressions; then applies `((x_i - mean) / sqrt(var + eps)) * gamma_i + beta_i` per column.\n- Output units stored under `orbital_ln_{lname}_h{i}` names.\n- **DAG trigger** in `orbital_keras_impl()` updated to route `layernorm` layers to the DAG path.\n\nIntegration test added: `keras3 Functional model with LayerNormalization translates correctly` — verifies `orbital_ln_mean_` and `orbital_ln_var_` intermediates appear in output.
